packages/aggregator/src/__tests__/analytics/aggregateAnalytics/aggregations/entityAggregations.test.js

```diff
@@ -25,18 +24,22 @@ const BASE_TEST_ANALYTICS = [
 describe('entityAggregations', () => {
   describe('sumPerOrgGroup()', () => {
     it('should do nothing without orgUnitMap', () => {
-      expect(sumPerOrgGroup(BASE_TEST_ANALYTICS, {})).to.have.same.deep.members([
-        { dataElement: 'element1', organisationUnit: 'org1', period: '20200101', value: 1 },
-        { dataElement: 'element1', organisationUnit: 'org2', period: '20200102', value: 2 },
-        { dataElement: 'element1', organisationUnit: 'org3', period: '20200103', value: 3 },
-      ]);
+      expect(sumPerOrgGroup(BASE_TEST_ANALYTICS, {})).toEqual(
+        expect.arrayContaining([
+          { dataElement: 'element1', organisationUnit: 'org1', period: '20200101', value: 1 },
+          { dataElement: 'element1', organisationUnit: 'org2', period: '20200102', value: 2 },
+          { dataElement: 'element1', organisationUnit: 'org3', period: '20200103', value: 3 },
+        ]),
+      );
```

Comment:
In the test cases of this file, we indeed cannot use `.toEqual` because the order of the results is not guaranteed. While the existing code is correct, it can be made even simpler be using `toIncludeSameMembers` from [jest-extended](https://github.com/jest-community/jest-extended#toincludeanymembersmembers) (already installed in our monorepo)

```suggestion
      expect(sumPerOrgGroup(BASE_TEST_ANALYTICS, {})).toIncludeSameMembers([
        { dataElement: 'element1', organisationUnit: 'org1', period: '20200101', value: 1 },
        { dataElement: 'element1', organisationUnit: 'org2', period: '20200102', value: 2 },
        { dataElement: 'element1', organisationUnit: 'org3', period: '20200103', value: 3 },
      ]);
```

Similarly for the other test cases in this file

Reply:
I can understand your confusion about `.toEqual(expect.arrayContaining())` here, which is exactly the same as `.toIncludesSameMembers`. Definely I will change it to `.toIncludesSameMembers()` because I agree it is hard to read.